packages/vite/package.json

```diff
@@ -115,7 +115,7 @@
     "source-map-support": "^0.5.20",
     "strip-ansi": "^6.0.0",
     "terser": "^5.9.0",
-    "tsconfck": "^1.0.0",
+    "tsconfck": "1.0.0",
```

Comment:
We should involve @dominikg to understand why we need this pinned to 1.0.0

Reply:
I think it's cause of these imports https://github.com/dominikg/tsconfck/compare/tsconfck%401.0.0...tsconfck@1.1.0#diff-4bd2be3447d4786be2b6777af66b599ae9868eb39a71bf881d43591b83a608ab

`import x from 'node:x'` seems not be supported by node v12

Reply:
It should be supported in 12.20 https://nodejs.org/api/esm.html#node-imports

tsconfck tests pass with node12 too.

Maybe it's an issue with vite bundling it?

Reply:
Why should it do this exactly for node v12? :thinking:

Reply:
Does it rewrite import to require?

Reply:
It looks like the cjs version of tsconfck contains code like this:
```
var import_node_path = __toModule(require("node:path"));
```
and `node:` prefix is not working for require in node12 :see_no_evil: 
i'll release a fix for that. Ideally esbuild would strip `node:` when converting to cjs for node12.

Reply:
But you could also wait on the decision next friday, whether we want to support v12 or not

Reply:
tsconfck is pretty low level and should continue to support node12 until it's end of life. Released 1.1.1 without 'node:' 
https://github.com/dominikg/tsconfck/blob/main/CHANGELOG.md#111-2021-10-31

Reply:
thanks for pinging me about it